lib/formats/api_blueprint.js

```diff
@@ -1,26 +1,22 @@
 'use strict';
 
+var Promise = require('bluebird');
 var Drafter = require('drafter.js');
 var Inherits = require('util').inherits;
 var BPToSwagger = require('apib2swagger');
 
-var BaseType = require('../base-type.js');
+var BaseFormat = require('../base_format.js');
 
 var APIBlueprint = module.exports = function() {
   APIBlueprint.super_.apply(this, arguments);
-  this.type = 'api_blueprint';
-
-  this.converters.swagger_2 = function(apibp, callback) {
-    try {
-      var swagger2 = BPToSwagger.convertParsed(apibp.spec);
-    } catch(e) {
-      return callback(e);
-    }
-    return callback(null, swagger2);
-  }
+  this.format = 'api_blueprint';
+
+  this.converters.swagger_2 = Promise.method(apibp => {
```

Comment:
Have we already been using ES6 syntax or will this introduce it?

As much as I love ES6, I have been trying to avoid it in any libraries I publish to remain compatible with node 3.x. I'm OK upgrading, but we'll probably get a few bug reports about syntax errors - should definitely add a note to the README at least